src/templates/emails/summary_sample_event.html

```diff
@@ -0,0 +1,12 @@
+{% extends "emails/base.html" %}
+{% block title %}Summary Sample Event Report{% endblock %}
+{% block body %}
+<h2>Summary Sample Event Report</h2>
+<p>
+    Your report is ready for download. Please click the link below to download the report.
+    
+    <strong><a href="{{file_url|safe}}">{{file_url|safe}}</a></strong>
+
+    <small>Link will be valid for 7 days.</small>
```

Comment:
Isn't it 14 days?

Reply:
We keep the file in S3 for 14 days but can only do pre-signed URL for a max of 7 days